frontend/src/app/history/download-history/download-history.component.html

```diff
@@ -7,6 +7,12 @@ <h1 class="title">Downloads</h1>
                 Downloads
             </div>
         </ng-template>
+        <ng-template *ngIf="downloads.length" pTemplate="summary">
+            <button class="button is-primary" type="reset" (click)="deleteAllDownloads()">
+                <span class="icon"><fa-icon [icon]="faTrash"></fa-icon></span>
+                <span>Delete all downloads</span>
+            </button>
+        </ng-template>
```

Comment:
I agree this is a good place for a delete button, but it's inconsistent that downloads are deleted here, while queries are deleted from the settings menu. The pages are almost identical, so they should handle this the same way. Perhaps you could move the "delete search history" button?